Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -1569,6 +1580,73 @@ - (IBAction)showImageChoices:(id)sender {
     [self beginTaskWithIdentifier:ImageChoicesTaskIdentifier];
 }
 
+# pragma mark - Image Capture
+- (id<ORKTask>)makeImageCaptureTask {
+    NSMutableArray *steps = [NSMutableArray new];
+    
+    {
+        ORKInstructionStep *step = [[ORKInstructionStep alloc] initWithIdentifier:@"begin"];
+        step.title = @"Hands";
+        step.image = [UIImage imageNamed:@"hands_solid"];
```

Comment:
shouldTintImages seemed like such an implementation detail of the framework that it seemed undesirable to expose it on ORKStep for general use, especially since in most apps you will already know what your tint color is.

My feeling is that ORKTintedImageView's shouldApplyTint property should not *override* the rendering mode of the input image to turn off tinting if it is already in templated rendering mode, which can only happen if someone has explicitly gone to the trouble of setting it. There are some funny details in ORKTintedImageView e.g. self.image = image to re-apply the tint, which would need to be resolved when changing this behavior.